module/algebras/user/src/main/scala/phms/algebra/user/UserAccountAlgebra.scala

```diff
@@ -1,18 +1,16 @@
-package pms.algebra.user
+package phms.algebra.user
 
-import doobie.util.transactor.Transactor
-import pms.algebra.user.impl.UserAlgebraImpl
-import pms.core.{ Async, Resource, _}
+import phms.algebra.user.impl.UserAlgebraImpl
+import phms._
+import phms.kernel._
+import phms.db._
 
-/**
-  *
-  * @author Lorand Szakacs, https://github.com/lorandszakacs
+/** @author Lorand Szakacs, https://github.com/lorandszakacs
   * @since 20 Jun 2018
-  *
   */
 trait UserAccountAlgebra[F[_]] {
 
-  implicit protected def monadError: MonadError[F, Throwable]
+  implicit protected def monadThrow: MonadError[F, Throwable]
```

Comment:
Maybe also change the type to `MonadThrow[F]` to be consistent with the rest of the changes?

Reply:
Thanks. Resolved.